plugins/unicodelib/unicodelib.cpp

```diff
@@ -719,6 +724,76 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+void normalizationFormCheck(UnicodeString & source, const char * form)
+{
+    UErrorCode errorCode = U_ZERO_ERROR;
+    if (form[2] == 'C')
+    {
+        const Normalizer2 * no = Normalizer2::getInstance(NULL, "nfc", UNormalization2Mode::UNORM2_DECOMPOSE, errorCode);
+        source = no->normalize(source, errorCode);
+    }
+    else if (form[2] == 'D')
+    {
+        const Normalizer2 * no = Normalizer2::getInstance(NULL, "nfc", UNormalization2Mode::UNORM2_DECOMPOSE, errorCode);
```

Comment:
I was following the ICU reference for the class.

"Use packageName=NULL for data files that are part of ICU's own data. Use name="nfc" and UNORM2_COMPOSE/UNORM2_DECOMPOSE for Unicode standard NFC/NFD. Use name="nfkc" and UNORM2_COMPOSE/UNORM2_DECOMPOSE for Unicode standard NFKC/NFKD. Use name="nfkc_cf" and UNORM2_COMPOSE for Unicode standard NFKC_CF=NFKC_Casefold."

http://icu-project.org/apiref/icu4c/classicu_1_1Normalizer2.html#a6e4ec68e4c05d4312195e77f0a743eb5